LoopKitTests/TherapySettingsTests.swift

```diff
@@ -176,6 +178,17 @@ class TherapySettingsTests: XCTestCase {
           }
         }
       },
+      "correctionRangeOverrides" : {
```

Comment:
This is one of my concerns using the `CorrectionRangeOverride` struct. It basically stores *one* unit field for *two* values/ranges. It would seem more complete to be something like:
```
        "preMealRange" : {
          "maxValue" : 90,
          "minValue" : 80,
          "bloodGlucoseUnit" : "mg/dL"
        },
        "workoutRange" : {
          "maxValue" : 140,
          "minValue" : 130,
          "bloodGlucoseUnit" : "mg/dL"
        }
```
or possible even:
```
        "preMealRange" : {
          "max" : {
             "value": 90,
             "unit": "mg/dL"
          },
          "max" : {
             "value": 80,
             "unit": "mg/dL"
          }
        },
        "workoutRange" : {
          "max" : {
             "value": 140,
             "unit": "mg/dL"
          },
          "max" : {
             "value": 130,
             "unit": "mg/dL"
          }
        }
```

Reply:
Yes, it is more "chatty", but also more complete and accurate.

Reply:
I like the first suggestion and will update like that.